src/content/references/en/6-ai-weather.md

```diff
@@ -0,0 +1,16 @@
+---
+image: ../../../images/references/ai-weather.webp
+title: AI pipeline for weather satellite data
+---
+
+Clouds in satellite imagery carry information about the weather, but can also block the view of the actual target. We developed an AI model as well as a matching data pipeline to detect and mask clouds. The preparation of the Meteosat Second Generation time series data used as well as the AI training and application are implemented as a repeatable workflow.
+
+## Highlights
+
+- Processing of very large weather satellite time series
+- Reusable pipeline for AI training/application
+- Training and application of Deep-Learning with Convolutional Neural Networks
+
+## Methodik
```

Comment:
should be Methodology

Reply:
[304b434](https://github.com/geo-engine/geoengine.de/pull/6/commits/304b4343dd4c0ea1ab9fdb71988815b9b1b5c145)